### DISCUSSION TITLE
AUX relay switches

### DISCUSSION THREAD

**Discussion by Karel311** [alexdelprete/HA-NeoPool-MQTT#13] [Q&A]
Hi Everyone,

i managed to install an atom lite+tail485 combination on my oxilife pool system  and the whole HA Neopool MQTT integration in HomeAssistant.

Only thing that is not working are the AUX relay swichtes in HA. 
Tasmota console gives a log message when trying to use any of the 4 aux switches in HA: 
`MQT: stat/SmartPool/RESULT = {"Command":"Unknown"}`

When switching AUX relays on the oxilife, i do see a change in the state in HA off the correct AUX relay.

Any ideas why they're not working as expected when controlled in HA?

**Comment by alexdelprete** [comment]
Did you add the user defined commands as described [here](https://tasmota.github.io/docs/NeoPool/#esp32-adding-user-defined-neopool-commands-to-tasmota)?

cc @curzon01

**Reply by Karel311** [reply]
:) Tx got it now

**Reply by alexdelprete** [reply]
Glad you solved it. As often happens, the solution is simple: RTFM! 🤣

**Comment by Karel311** [comment]
Other related question: 
flow switch of the ph electrode has state ON  when pool pump is of. 

Is there a way to create an inverted switch with ON state when pool pump is running ?   

Already tried to create this, but only has state "true"or "false", so that the icons don't change color when state changes: 

`      inverted_neopool_mqtt_ph_fl1:
friendly_name: "Flowsensor pH elektrode"
value_template: "{{ is_state('binary_sensor.neopool_mqtt_ph_fl1', 'off') }}"`

**Comment by alexdelprete** [comment]
Please open a separate issue for the flow swith. I ask you to do it because I might have found a missing binary_sensor, so I want to investigate it separately.

Please close this one and open another issue. Thanks.

**Comment by Karel311** [comment]
Solved by Alex
